openmdao/docs/examples/hohmann_transfer/hohmann_transfer.rst

```diff
@@ -0,0 +1,230 @@
+   :align: center
+   :width: 500 px
+   :alt: Vector diagram of the second impulsive :math:`\Delta V`
+
+   Vector diagram of the second impulsive :math:`\Delta V`
+
+.. math::
+
+    \Delta V = \sqrt{ v_a^2 + v_c^2 - 2 v_a v_c \cos{\Delta i} }
+
+.. math::
+
+    v_a = \frac{h}{r_a}
+
+Having already computed the specific angular momentum of the transfer orbit, this is
+easily computed.
+
+Finally we have the necessary calculations to compute the :math:`\Delta V` of the Hohmann
+transfer with a plane change.
+
+
+~~~~~~~~~~
+Components
+~~~~~~~~~~
+
+The first component we define computes the circular velocity given the
+radius from the center of the central body and the gravitational parameter
+of the central body.
+
+.. literalinclude:: ../../../test_suie/test_examples/test_hohmann_transfer.py
+   :pyobject: VCircComp
+
+The transfer orbit component computes the velocity magnitude at periapsis
+and apoapsis of an orbit, given the radii of periapsis and apoapsis, and
+the gravitational parameter of the central body.
+
+.. literalinclude:: ../../../test_suie/test_examples/test_hohmann_transfer.py
+   :pyobject: TransferOrbitComp
+
+The delta-V component is used to compute the delta-V performed in changing
+the velocity vector, giving the magnitudes of the initial and final velocities
+and the angle between them.
+
+.. literalinclude:: ../../../test_suie/test_examples/test_hohmann_transfer.py
+   :pyobject: DeltaVComp
+
+~~~~~~~~~~~~~~~~~~~~~~~
+Putting it all together
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we assemble the model for our problem
+
+An IndepVarComp provides values for the gravitational parameter, the radii
+of the two circular orbits, and the delta-V to be performed at each of the
+two impulses.
+
+Next, two instances of VCircComp are used to compute the velocity of the
+spacecraft in the initial and final circular orbits.
+
+The TransferOrbitComp is used to compute the periapsis and apoapsis velocity
+of the spacecraft in the transfer orbit.
+
+Now we can use the DeltaVComp to provide the magnitude of the delta-V
+at each of the two impulses.
+
+Lastly, we use two ExecComps to provide some simple calculations.  One
+sums the delta-V's of the two impulses to provide the total delta-V of the
+transfer.  We will use this as the objective for the optimization.
+
+The other ExecComp sums up the inclination change at each impulse.  We
+will provide this to the driver as a constraint to ensure that our total
+inclination change meets our requirements.
+
+We will use the initial and final radii of the orbits, and the inclination
+change at each of the two impulses as our design variables.
+
+To run the model we provide values for the design variables and invoke `run_model`.
+
+To find the optimal solution for the model, we invoke `run_driver`, where we have
+defined the driver of the problem to be the ScipyOptimizer.
+
+.. embed-test::
+    openmdao.test_suite.test_examples.test_hohmann_transfer.TestHohmannTransfer.test_dv_at_apogee
```

Comment:
I think the Hohmann transfer example is missing from the repo.

Reply:
@Kenneth-T-Moore the tests would concur with you:
```Exception occurred:
  File "C:\Miniconda-x64\lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named test_examples.test_hohmann_transfer